eos-updater-prepare-volume/eos-updater-prepare-volume

```diff
@@ -213,11 +216,15 @@ def main():
                     'used to update other machines offline. The repository '
                     'copy will be put in the .ostree/repo directory on the '
                     'USB drive; other files will not be affected.')
-    parser.add_argument('volume_path', metavar='VOLUME-PATH', type=str,
-                        help='path to the USB drive to prepare')
     parser.add_argument('--quiet', action='store_const', const=True,
                         help='do not print anything; check exit status '
                              'for success')
+    parser.add_argument('--preserve-permissions', dest='preserve_permissions',
```

Comment:
New option needs to be documented in `eos-updater.git/eos-updater-prepare-volume/docs/eos-updater-prepare-volume.8`.